site/content/docs/user/local-registry.md

```diff
@@ -28,4 +28,4 @@ The registry can be used like this.
 4. And now we can use the image `kubectl create deployment hello-server --image=localhost:5001/hello-app:1.0`
 
 If you build your own image and tag it like `localhost:5001/image:foo` and then use
-it in kubernetes as `localhost:5001/image:foo`.
+it in kubernetes as `localhost:5001/image:foo`. And use it from inside of your cluster application as `${reg_name}:5000`.
```

Comment:
since ${reg_name} is fixed in the script, maybe it would be clearer to say that?

Reply:
for sure updated